src/components/TodoList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TodoItem from './TodoItem';
+
+class TodoList extends React.PureComponent {
+  render() {
+    const { todos } = this.props;
+    const { handlechangeProps } = this.props;
+    const { handledeleteProps } = this.props;
```

Comment:
- [ ]  You can join this 3 const in one line like this: const `{ todo, handlechangeProps, handledeleteProps } = this.props;`

Reply:
Thank you @ryxtor , you are right, it will make the code shorter and easy to understand.👌